sample-app/src/components/DropdownSection.tsx

```diff
@@ -37,8 +43,28 @@ export default function DropdownSection({
 }: DropdownSectionProps): JSX.Element | null {
 
   const [focusedOptionIndex, setFocusedOptionIndex] = useState<number>(0);
+  const [focusedLinkIndex, setFocusedLinkIndex] = useState<number>(-1);
+  const browserHistory = useHistory();
   const wasFocused = useRef<boolean>(isFocused);
 
+  function updateToNextFocusElement() {
+    if(!incrementVerticalLinkFocus()) {
+      incrementOptionFocus();
+    }
+  }
+
+  function incrementVerticalLinkFocus(): boolean {
+    const verticalLinks = options[focusedOptionIndex]?.verticalLinks;
+    if (verticalLinks) {
```

Comment:
nit: I have a preference for returning early so that the code isn't nested so much. For example, we could have `if (!verticalLinks) { return false; }`, and then the rest of the function would follow without needing to be nested

Reply:
I see, will update!